packages/components/src/components/buttons/Button/index.native.tsx

```diff
@@ -2,42 +2,95 @@ import React from 'react';
 import styled, { css } from 'styled-components/native';
 import { getPrimaryColor, getSecondaryColor } from '../../../utils/colors';
 import colors from '../../../config/colors';
-import { Omit, FeedbackType } from '../../../support/types';
+import { Omit, FeedbackType, IconShape } from '../../../support/types';
+import Icon from '../../Icon';
+import { FONT_SIZE_NATIVE } from '../../../config/variables';
 
 const Spinner = styled.ActivityIndicator`
     padding-right: 10px;
 `;
 
-const View = styled.View`
+const Wrapper = styled.View`
+    flex: 1;
     flex-direction: row;
+    align-items: center;
+    justify-content: center;
 `;
 
-interface LabelProps {
+interface LabelProps extends Omit<Props, 'onClick'> {
     disabled: boolean;
 }
 const Label = styled.Text<LabelProps>`
     font-weight: 400;
     align-self: center;
-    font-size: 18;
-    color: ${(props: any) => (props.disabled ? colors.TEXT_SECONDARY : colors.WHITE)};
+    color: ${colors.WHITE};
+    font-size: ${FONT_SIZE_NATIVE.BASE};
+
+    ${(props: any) =>
```

Comment:
why `any`? types for props are explicitly declared in `styled.Text<LabelProps>`, if `LabelProps` contains `isInverse `, `disabled`,... then it should just work without `any` hack?

Reply:
fixed